MediaStore/app/src/main/res/values/strings.xml

```diff
@@ -24,4 +24,9 @@
     <string name="grant_permission">Grant Permission</string>
 
     <string name="permission_not_granted">App requires access to storage to access items stored in MediaStore.</string>
+
+    <string name="delete_title">Delete?</string>
```

Comment:
Nit: `delete_dialog_title`. Same prefix for the rest of the strings too.

Reply:
Done.